modules/acquia_cms_common/acquia_cms_common.services.yml

```diff
@@ -45,14 +45,3 @@ services:
       - '@theme_handler'
       - '@string_translation'
       - '@extension.list.module'
-  acquia_cms_common.telemetry:
-    class: Drupal\acquia_cms_common\EventSubscriber\KernelTerminate\AcquiaCmsTelemetry
-    arguments:
-      - '@extension.list.module'
-      - '@config.factory'
-      - '@state'
-      - '%site.path%'
-      - '@datetime.time'
-      - '@logger.factory'
-    tags:
-      - { name: event_subscriber }
```

Comment:
Removal of acquia_cms_common.telemetry will break the BC.
cc @vishalkhode1 @chandan-singh7929

Reply:
I don't see any issue with this.